docs/controls/navigation-control.md

```diff
@@ -0,0 +1,25 @@
+# Navigation Control
+
+This is a React equivalent of Mapbox's [NavigationControl](https://www.mapbox.com/mapbox-gl-js/api/#navigationcontrol):
+
+```js
+import ReactMap, {Marker} from 'react-map-gl';
+
+/* Needs Mapbox's stylesheet to work! */
+import 'mapbox-gl/dist/mapbox-gl.css';
```

Comment:
You will need to add configs to either bundler if you want this to work
I added style-loader and css-loader to the webpack config in the example app.
Browserify has a css transform as well.
I will add more comments here.